module/database/variables.tf

```diff
@@ -0,0 +1,47 @@
+variable "zone" {
```

Comment:
For the default all the things, you must imagine that someone will come, just copy/paste the database folder and try to add it in his codebase. 
If you have default everywhere it will be really easy.